charts/kserve-resources/templates/deployment.yaml

```diff
@@ -25,7 +25,7 @@ spec:
       serviceAccountName: kserve-controller-manager
       containers:
       - name: kube-rbac-proxy
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.4.0
```

Comment:
@ddelange Can you also help update the version in helm chart?
https://github.com/kserve/kserve/blob/master/charts/kserve-resources/templates/deployment.yaml#L28

Reply:
which version do you mean to update exactly? the `Chart.yaml` version bump is part of your release process I thought? so I didn't touch it. Would you like me to increase it to rc2?

Reply:
@ddelange I meant for making the same change for `kube-rbac-proxy` version in the helm chart not the chart version.

Reply:
We have two installation methods, the place you modified is for kustomize and we also support helm installation.

Reply:
> the line you posted is the line which this PR updates

this PR updates the tag in the helm chart, can you link me to the line which additionally needs updating?

Reply:
@ddelange sorry my bad to give the wrong link, it should be this place 
https://github.com/kserve/kserve/blob/master/config/default/manager_auth_proxy_patch.yaml#L13

Reply:
do you mean https://github.com/kserve/kserve/blob/19ee4cf428f852007b3ca979029de209fbf88ab9/config/default/manager_auth_proxy_patch.yaml#L13 ?

Reply:
done!

Reply:
ah damn DCO, github web ui didnt recognise it, sec